packages/backend/src/util/environment.ts

```diff
@@ -1,3 +1,13 @@
-import * as env from 'env-var'
+import { config } from 'dotenv'
+import { from } from 'env-var'
+const env = from(process.env, {})
+config()
 
-export const BACKEND_PORT = env.get('BACKEND_PORT').default(3300).asInt()
```

Comment:
We could wrap these config variables into one config object. That would make them a bit more organized
Also creating an object allows to use Nest Injection model to inject the config into services.

This is not a huge deal, rather than an advice

Reply:
This topic ín Issue #23 I thin

Reply:
Nyeh, I see. Still think this is not the prettiest solution, but if this is in the ticket go for it.
Othervise good PR, with clear to understand commit messages, and description

Reply:
Thanks